src/clocksi_static_tx_coord_sup.erl

```diff
@@ -28,7 +28,7 @@
 
 -export([init/1]).
 
--define(NUM, 100).
+-define(NUM, 1).
```

Comment:
Again, as I've previously stated, I'm not going to merge a change that shows a number go up without knowing *why* the change is triggering that.  You still have not justified *why* the throughput is changing. 

Can you explain why changing: 

1 - 5: causes decrease in throughput
5 - 10: causes increase in throughput
10 - 20: decrease
etc.

We need to know what is really happening.